controllers/keystoneapi_controller.go

```diff
@@ -204,27 +214,26 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 		},
 	)
 	// create or patch the DB
-	cond, ctrlResult, err := db.CreateOrPatchDB(
+	ctrlResult, err := db.CreateOrPatchDB(
 		ctx,
 		helper,
 	)
-	instance.Status.Conditions.UpdateCurrentCondition(cond)
-
+	//instance.Status.Conditions.Set(condition.FalseCondition(condition.DBReadyCondition,db.GetDatabase().Status))
 	if err != nil {
+		//c := condition.FalseCondition(con)
+		//instance.Status.Conditions.Set(c)
```

Comment:
why it is comment out? it make sens to set a False conditions here

Reply:
It does seem like this database logic should follow the same condition patterns as used for `DBSync`, `ExposeServices`, etc.

Reply:
This part is right now commented out as I wanted for this just to sync the condition on the mariadbdatabase object when that mariadb-operator controller reconciles and updates it. this would first required to get the new pattern into that operator. Until that is done I think I'll finis this part here to follow the pattern used for DBSync, ...

Reply:
done

Reply:
ack. make sense